.github/workflows/preview.yml

```diff
@@ -97,7 +97,7 @@ jobs:
 
       - name: Publish to TestPyPI
         if: inputs.TestPyPI
-        run: poetry publish --repository testpypi --username __token__ --password ${{ secrets.TESTPYPI_API_TOKEN }}
+        run: poetry publish -vvv --repository testpypi --username __token__ --password ${{ secrets.TESTPYPI_API_TOKEN }}
```

Comment:
Even if we're not logging this to the GitHub logs, we should avoid potentially leaking tokens to other processes via command line arguments.

Reply:
Good idea. The suggestion has been added to this review.